packages/server/integrations/jiraServer/JiraServerRestManager.ts

```diff
@@ -46,7 +72,34 @@ export default class JiraServerOAuth1Manager {
     return response
   }
 
-  async getProjects() {
-    return this.request('GET', '/rest/api/latest/project')
+  async parseJsonResponse(response: Response) {
```

Comment:
+1 I really like this!
May be convenient to make this a generic so it doesn't return an implicit `any` & we don't need to explicitly declare a return type on L90.

e.g.

```ts
  const projects = this.parseJsonResponse<JiraServerRestProject[]>(response) // Error | JiraServerRestProject[]
```